src/movement/control_system.cpp

```diff
@@ -38,20 +29,13 @@ namespace robosub
                 r_lim), "Failed to load the rotiation control limit.");
         ROS_ERROR_COND(!ros::param::getCached("/control/max_thrust",
                 max_thrust), "Failed to load the max thrust output.");
-        ROS_ERROR_COND(!ros::param::getCached("/control/rate", rate),
-                "Failed to load the control system rate.");
 
         /*
-         * Calculate the change in time between each call.
-         * TODO: Replace dt with actual message timestamps.
+         * All translational masses are the submarines total mass.
          */
-        if (rate <= 0)
-        {
-            ROS_FATAL("Control system rate is specified as zero or less.");
-            exit(-1);
-        }
-
-        dt = 1.0/rate;
+        ROS_ERROR_COND(!ros::param::getCached("/control/mass", sub_mass[0]),
```

Comment:
remove the leading '/' on the parameter, we don't want to be using absolute paths.